cmd/juju/application/bundle.go

```diff
@@ -119,6 +121,7 @@ func deployBundle(
 	bundleDevices map[string]map[string]devices.Constraints,
 	dryRun bool,
 	force bool,
+	trust bool,
 	useExistingMachines bool,
 	bundleMachines map[string]string,
 ) (map[*charm.URL]*macaroon.Macaroon, error) {
```

Comment:
So having spoken to @jameinel recently about this, we should make this series of arguments a struct. It's becoming clear that this is overloaded.

Reply:
I also thought about this when I was going through the code. I will push an extra commit to address this.